### DISCUSSION TITLE
How can I configure a new cluster on the Mia-Platform Console?

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#90] [Q&A] [answered]
Hi!

I am trying to create a new Environment for my project (v10.0.1) and I want to put it on a brand new Kubernetes cluster. The cluster has already been created. How can I make the cluster appear in the `Cluster Id` select box in the screenshot below?

<img width="1214" alt="Screenshot 2022-11-10 at 14 47 24" src="https://user-images.githubusercontent.com/5745481/201108385-c65512e0-6dfc-4c92-a494-b8ad16ab4358.png">

Thanks!

**Comment by frengooo** [comment, marked as answer]
Hi @Lucascanna,
I think you need to add the new cluster into the dedicated section.
Go into the "Company" section and, in the side menu, you can find the "Clusters" page:
<img width="1920" alt="Screenshot 2022-11-10 alle 15 01 57" src="https://user-images.githubusercontent.com/28152586/201111974-9d87710f-d9a6-43c1-906b-39e40d16657f.png">

You can manage the clusters of your company with the v10.
Check the public documentation for more:
https://docs.mia-platform.eu/docs/development_suite/clusters-management
